modules/cache/jvm/src/main/scala/coursier/cache/CacheUrl.scala

```diff
@@ -142,7 +142,7 @@ object CacheUrl {
 
         // Early in the development of coursier, I ran into some repositories (Sonatype ones?) not
         // returning the same content for user agent "Java/…".
-        conn0.setRequestProperty("User-Agent", "")
+        conn0.setRequestProperty("User-Agent", "Coursier")
```

Comment:
Let's just remove this line (and the comment above), the default user agent (something like `Java/…`) should be fine.

Reply:
Done.